### PR TITLE
tasks/20119 Filling layout

### PR DESCRIPTION
**Pull request by TranDangSy**
==> Title: refers tasks/20119 Filling layout 

==> Redmine ticket: https://edu-redmine.sun-asterisk.vn/issues/20119

==> Attachment: 
image

### CODE REVIEW

**Comment by TranDangSy** [comment]
ready

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<footer id="footer">
+<div class="footer-top">
```
Comment (ngocvt-0484) [inline app/views/layouts/_footer.html.erb]: ident e ơi

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,127 @@
+<header id="header">
+<div class="header_top">
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb]: ident

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            <% end %>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="header-bottom">
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb]: view dài quá tách view ra nha e

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -5,4 +5,10 @@
     <%= render "layouts/shim" %>
   </head>
   <body>
+    <%= render "layouts/header" %>
+    <div class="container">
+      <%= yield %>
+    <%= render "layouts/footer" %>
```
Comment (ngocvt-0484) [inline app/views/layouts/application.html.erb]: ident

File: app/views/products/home.html.erb [hunk 5]
```diff
@@ -0,0 +1,125 @@
+<section id="slider">
+<div class="container">
```
Comment (ngocvt-0484) [inline app/views/products/home.html.erb]: (vả) lại ident

File: app/views/products/home.html.erb [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+                <ul>
+                  <li><%= link_to t("home.text.alarm"), "#" %></li>
+                  <li><%= link_to t("home.text.timekeeper"), "#" %></li>
+                  <li><%= link_to t("home.text.journey"), "#" %></li>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="brands_products">
```
Comment (ngocvt-0484) [inline app/views/products/home.html.erb]: view dài tách view e ơi

**Comment by TranDangSy** [comment]
ready

**Comment by TranDangSy** [comment]
ready

**Comment by TranDangSy** [comment]
ready

**Pull request merged**